dotnetv3/Glacier/ListJobsExample/ListJobs.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// List the Amazon Glacier jobs for a vault. This example was created using
+/// the AWS .NET SDK version 3.7 and .NET Core 5.0.
+/// </summary>
+namespace ListJobsExample
+{
+    // snippet-start:[Glacier.dotnetv3.ListJobsExample]
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon.Glacier;
+    using Amazon.Glacier.Model;
+
+    class ListJobs
+    {
+        static async Task Main(string[] args)
+        {
+            var client = new AmazonGlacierClient();
+            var vaultName = "example-vault";
+
+            var request = new ListJobsRequest
+            {
+                AccountId = "-",
```

Comment:
Is the dash character valid? If not, do you want to follow the other model and make is something like "example-account"?

Reply:
This is valid. If you use a - for AccountID, it uses the account associated with the default user. I added a comment explaining that.